pkg/operator2/configobservation/console/observe_consoleurl.go

```diff
@@ -0,0 +1,43 @@
+package console
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	"github.com/openshift/cluster-authentication-operator/pkg/operator2/configobservation"
+)
+
+func ObserveConsoleURL(genericlisters configobserver.Listers, recorder events.Recorder, previouslyObservedConfig map[string]interface{}) (map[string]interface{}, []error) {
```

Comment:
I think we should add `pruning` as we did in https://github.com/openshift/library-go/pull/744

Reply:
correct, I forgot about that, this code predates that change